docs/Running-Mastodon/Production-guide.md

```diff
@@ -200,7 +199,7 @@ After=network.target
 Type=simple
 User=mastodon
 WorkingDirectory=/home/mastodon/live
-Environment="NODE_ENV=production"
+Environment="RAILS_ENV=production"
```

Comment:
NODE_ENV != RAILS_ENV, this is a Node application, so you don't need to set RAILS_ENV here